specification/app/resource-manager/Microsoft.App/preview/2023-11-02-preview/Builds.json

```diff
@@ -38,7 +38,7 @@
         "tags": [
           "Builds"
         ],
-        "operationId": "Builds_ListByBuilderResource",
+        "operationId": "BuildsByBuilderResource_List",
```

Comment:
I'm good with the change, but I'm confused by the initial error. :) It says that:
![image](https://github.com/Azure/azure-rest-api-specs/assets/48293037/831a3281-da43-4cb7-af5a-1ea8805c8c69)

However here "Builds" was never after the underscore. Am I missing something?

Reply:
It should be `Noun_Verb` so Resource_Action in this case

Refer https://github.com/Azure/azure-openapi-validator/blob/main/docs/operation-id-noun-verb.md

Reply:
Just a bit of context here, we originally generated these definitions using [ARM TypeSpec](https://armwiki.azurewebsites.net/rp_onboarding/process/onboarding.html#1-review-the-apis), which is now being enforced for new RPs (which we originally were planning on creating). The operation ID that typespec generates for child resources is `Child_ListByParentResource` (generated the below screenshot from the playground[ linked from their docs](https://eng.ms/docs/products/azure-developer-experience/design/api-specs/api-typespec))

![image](https://github.com/Azure/azure-rest-api-specs/assets/107068277/5ad365e1-3dea-4a47-8285-fa0c8f03bc86)

We setup [a rule to supress this warning](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/app/resource-manager/readme.md?plain=1#L64-L68), since it only appears because `Builder` has `Build` in the name, so this error is incorrectly displayed here